### PR TITLE
Add Tools/GPU-Occupancy-Calculator/index.html via file copy rather than network download

### PR DESCRIPTION
**Pull request by pdxjohnny**
Also [enables testing on pull request of github pages build](https://github.com/oneapi-src/oneAPI-samples/pull/1572/commits/ac90891613646380a80ddc93b05383f9d62c76c8)

### CODE REVIEW

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -50,9 +50,8 @@ jobs:
       env: 
         GPU_OCC_CALC: src/docs/_build/Tools/GPU-Occupancy-Calculator/
       run: |
-        mkdir -p ${GPU_OCC_CALC}   
-        cd ${GPU_OCC_CALC}
-        wget https://raw.githubusercontent.com/oneapi-src/oneAPI-samples/master/Tools/GPU-Occupancy-Calculator/index.html
+        mkdir -p ${GPU_OCC_CALC}
+        cp -v ${{ github.workspace }}/master/Tools/GPU-Occupancy-Calculator/index.html ${GPU_OCC_CALC}/index.html
```
Comment (mvincerx) [inline .github/workflows/github-pages.yml:55]: @pdxjohnny -- I'm fine with using this method by using github context variable (`github.workspace`).

First I need to know if you have tested it. ?

Comment (pdxjohnny) [reply .github/workflows/github-pages.yml:61]: Looks like we are good to go :)

```
'/home/runner/work/oneAPI-samples/oneAPI-samples/master/Tools/GPU-Occupancy-Calculator/index.html' -> 'src/docs/_build/Tools/GPU-Occupancy-Calculator//index.html'
```

Comment (mvincerx) [reply .github/workflows/github-pages.yml:61]: IM me asap, pls. 😺

Comment (mvincerx) [reply .github/workflows/github-pages.yml:55]: Incorporated 2 lines from this PR in https://github.com/oneapi-src/oneAPI-samples/pull/1585. 

You may now close this PR. Thanks for the contrib.

**Pull request closed without merging**